pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/util/internal/pex/PipFreezeAction.java

```diff
@@ -50,7 +50,14 @@ public PipFreezeAction(Project project) {
         developmentDependencies.addAll(configurationToSet(project, PythonPlugin.CONFIGURATION_TEST));
 
         // Special cases, such as sphinx-rtd-theme with weird metadata
-        developmentDependencies.addAll(Arrays.asList("sphinx-rtd-theme", "sphinx_rtd_theme"));
+        developmentDependencies.addAll(Arrays.asList(
```

Comment:
I have an algorithm ready for testing, but we need this fix to merge Ethan's branch internally.
I'll add a better fix soon.